Bottom: fc1bfdbd128be2dbba6b347418f7877fb7fd4bd4
Top:    fce03408593e9288463a098c5ffd85ec722c5ea6
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-02-10 19:11:37 +0000

Refresh of use-rtree-of-countours-when-co

---

diff --git a/src/polygon1.c b/src/polygon1.c
index 320ea18..416e092 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -846,8 +846,6 @@ intersect (jmp_buf * jb, POLYAREA * b, POLYAREA * a, int add)
   setjmp (c_info.restart);		/* we loop back here whenever a vertex is inserted */
 #endif
 
-//  printf ("Done making r-tree of b's contours\n");
-
   for (pa = a->contours; pa; pa = pa->next)     /* Loop over the contours of POLYAREA "a" */
     {
       BoxType sb;
@@ -864,7 +862,6 @@ intersect (jmp_buf * jb, POLYAREA * b, POLYAREA * a, int add)
             {
               /* The intersection test short-circuited back here,
                * we need to clean up, then longjmp to jb */
-//                printf ("short circuited here\n");
               r_destroy_tree (&b_contour_tree);
               longjmp (*jb, retval);
             }
